src/Command/ImportExisting.php

```diff
@@ -0,0 +1,309 @@
+<?php
+
+namespace App\Command;
+
+use App\Entity\Takedown\Takedown;
+use App\Entity\Takedown\ChildProtection\ChildProtection;
+use App\Entity\Takedown\Dmca\Dmca;
+use App\Util\TakedownUtilInterface;
+use Doctrine\Bundle\DoctrineBundle\Registry;
+use Doctrine\DBAL\Connection;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ImportExisting extends Command {
+
+	/**
+	 * @var Registry
+	 */
+	protected $doctrine;
+
+	/**
+	 * @var Connection
+	 */
+	protected $old;
+
+	/**
+	 * @var TakedownUtilInterface
+	 */
+	protected $takedownUtil;
+
+	/**
+	 * @var string[]
+	 */
+	protected const SITE_IDS = [
+		'en.wikipedia.org' => 'enwiki',
+		'wikimediafoundation.org' => 'foundationwiki',
+		'cy.wikipedia.org' => 'cywiki',
+		'en.wiktionary.org' => 'enwiktionary',
+		'commons.wikimedia.org' => 'commonswiki',
+	];
+
+	/**
+	 * @var string[]
+	 */
+	protected const PROJECT_IDS = [
+		'Wikimedia Commons' => 'commonswiki',
+	];
+
+	/**
+	 * @var string[]
+	 */
+	protected const USER_NAMES = [
+		'kbrown' => 'Kbrown (WMF)',
+		'ktsouroupidou' => 'Kalliope (WMF)',
+	];
+
+	/**
+	 * @var string[]
+	 */
+	protected const APPROVER_NAMES = [
+		'Geoff Brigham' => 'GeoffBrigham (WMF)',
+		'Geoff' => 'GeoffBrigham (WMF)',
+		'Jacob Rogers' => 'Jrogers (WMF)',
+		'Jacob' => 'Jrogers (WMF)',
+		'Luis Villa' => 'LuisV (WMF)',
+		'Luis' => 'LuisV (WMF)',
+		'Stephen LaPorte' => 'Slaporte (WMF)',
+		'Stephen' => 'Slaporte (WMF)',
+	];
+
+	/**
+	 * @var string[]
+	 */
+	protected const METADATA_IDS = [
+		'Checkuser data was available and is being included below.' => 'checkuser',
+		// @codingStandardsIgnoreLine
+		'An email was sent to legal@rt.wikimedia.org with the file name asking for it to be deleted.' => 'email-request',
+		// @codingStandardsIgnoreLine
+		'The content was taken down and we have awareness of facts or circumstances from which infringing activity is apparent. ' => 'taken-down-apparent',
+		'The content was taken down pursuant to a DMCA notice.' => 'taken-down-dmca',
+		// @codingStandardsIgnoreLine
+		'The content was taken down and we have actual knowledge that the content was infringing copyright ' => 'taken-down-infringing',
+		'The content was taken down and suppressed.' => 'taken-down-suppressed',
+		// @codingStandardsIgnoreLine
+		'The content was taken down and the user was clearly warned and discouraged from future violations.' => 'taken-down-user-warned',
+		'The user who uploaded the content has been locked.' => 'user-locked',
+	];
+
+	/**
+	 * {@inheritdoc}
+	 *
+	 * @param Registry $doctrine Doctrine.
+	 * @param Connection $old Old Database Connection.
+	 * @param TakedownUtilInterface $takedownUtil Takedown Utility.
+	 */
+	public function __construct(
+		Registry $doctrine,
+		Connection $old,
```

Comment:
$old is not very descriptive

Reply:
I have  a description in the doc bloc, is the old database connection. This script wont be used after the one-time import.